pkg/service/replay/absmatch.go

```diff
@@ -0,0 +1,454 @@
+package replay
```

Comment:
`match.go` is already a big file, thought of moving to `match.go` but here this abs match is not related to anything but test-bench matching. It will make it harder to read as the file will become too big.